### PR TITLE
Chat improvment

### PR DESCRIPTION
**Pull request by haimroizman**

### CODE REVIEW

File: DB/chat_repository.py [hunk 1]
```diff
@@ -1,22 +1,25 @@
...
 class ChatMessageRepository(Repository):
     def __init__(self):
         Repository.__init__(self, 'mysql_chat_db')
 
     def insert_message_execute(self, **kwargs):
-        sql_query = "INSERT INTO MESSAGE(message_col,user_id ) VALUES('%s','%s')" % (
-            kwargs['message_col'], kwargs['user_id'])
+        sql_query = "INSERT INTO MESSAGE(message_col,user_id,send_date ) VALUES('%s','%s','%s')" % (
+            kwargs['message_col'], kwargs['user_id'], kwargs['send_date'])
         self.execute_query(sql_query)
```
Comment (eranbetzalel) [inline DB/chat_repository.py]: http://stackoverflow.com/a/775399

File: DB/config.ini [hunk 2]
```diff
@@ -1,5 +1,5 @@
 [mysql_chat_db]
-host = 127.0.0.1
+host = 10.0.2.3
 database = chat
 user = root
-password = h1r2F3l4
```
Comment (eranbetzalel) [inline DB/config.ini]: I can still see it...

File: DB/chat_repository.py (hunk 1 above)
Comment (haimroizman) [inline DB/chat_repository.py]: done Mr

File: DB/config.ini (hunk 2 above)
Comment (haimroizman) [inline DB/config.ini]: Now you don't, in 2 minutes you won't

**Pull request merged**